pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1526,8 +1526,8 @@ def apply(self):
 
         # if installation media or hdd aren't used and settings have changed
         # try if source is available
-        if self.data.method.method not in ["cdrom", "harddrive"] and self.networking_changed:
```

Comment:
I have removed this check because it's already contained in needsNetwork method and it also skipping additional repositories now.